features/worldwide-organisations.feature

```diff
@@ -53,25 +53,6 @@ Feature: Administering worldwide organisation
     When I choose "Head office" to be the main office
     Then the "Head office" should be marked as the main office
 
-  Scenario: Adding default access information to a worldwide organisation
```

Comment:
these tests will start failing when you remove the access and opening tab from the subnav so we can just mark them as 

```

  @bootstrap-only
```

Reply:
done...